fastscape/tests/test_processes_tectonics.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+
+import pytest
+
+from fastscape.processes import (BlockUplift,
+                                 HorizontalAdvection,
+                                 SurfaceAfterTectonics,
+                                 TectonicForcing,
+                                 TwoBlocksUplift)
+from fastscape.processes.context import FastscapelibContext
+
+
+def test_tectonic_forcing():
```

Comment:
Rather than arrays with random values, it would be better to use arrays with uniform values. It allows to:

- hard-code expected values (no need / not a good practice to re-implement the logic actually tested in the test function)
- easier to see what's going wrong in case where the test fails.

Also, it would be nice to test the case where no or scalar values are given in the forcing var lists (to check that `p.bedrock_upward` and/or `p.surface_upward` correctly return a scalar value or zero).